debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -2157,7 +2157,239 @@ access_driver_type
 
 external_table_data_props
     : DEFAULT DIRECTORY quoted_string
-    | LOCATION
+    | ACCESS PARAMETERS '(' et_oracle_loader | et_oracle_datapump | et_oracle_hdfs_hive | USING CLOB subquery ')'
+    | LOCATION '(' et_directory_spec? et_location_specifier (',' et_directory_spec? et_location_specifier )* ')'
+    ;
+
+et_oracle_loader
```

Comment:
Wow, did you create all these rules from scratch, @ani-sha?

Reply:
Ya @gunnarmorling we had a bit of a journey discovering all these rules for all of Oracle's data loader tools.